docs/guide/testrunner/cloudservices.md

```diff
@@ -0,0 +1,145 @@
+name: cloudservices
+category: testrunner
+tags: guide
+index: 2
+title: WebdriverIO - Using Cloud Services
+---
+
+# Using Cloud Services
+
+Using cloud services with WebdriverIO is pretty simple.
+
+The two basic steps are:
+
+1. Make sure that Sauce uses this host for as the selenium server.
+2. (optional) set service specific values on `desiredCapabilities`.
+3. (optional) tunnell local trafic to provider, so that your tests can access
+   `localhost`
+
+## [Sauce Labs](https://saucelabs.com/)
+
+It is easy to set up your tests to run remotely in Sauce Labs.
+
+The only requirement is to set the `user` and `key` in your config (either
+exported by `wdio.conf.js` or passed into `webdriverio.remote(...)`) to your
+Sauce Labs username and access key.
+
+You can also pass in any optional [test configuration option](https://docs.saucelabs.com/reference/test-configuration/#webdriver-api)
+as a key/value in the capabilities for any browser.
+
+### [Sauce Connect](https://docs.saucelabs.com/reference/sauce-connect/)
+
+If you want to run tests against a server that is not accessible to the
+internet (like on `localhost`), then you need to use Sauce Connect.
+
+It is out of the scope of WebdriverIO to support this, so you must start it
+yourself.
+
+**Note:** Some people have reported trouble connecting to local websocket
+connection over Sauce Connect. This does work with BrowserStack and we
+reccommend you
+
+### With Travis CI
+
+Travis CI, however, does
+[have support](http://docs.travis-ci.com/user/sauce-connect/#Setting-up-Sauce-Connect)
+for starting Sauce Connect before each test, so follow their directions
+for that if you are interested.
+
+If you do so, you must set the `tunnel-identifier` test configuration option
+in each browser's capabilities. Travis sets this to the `TRAVIS_JOB_NUMBER`
+environmental variable by default.
```

Comment:
could you add an example/sentence like
```js
'tunnel-identifier': process.env.TRAVIS_JOB_NUMBER,
```
so the user knows that the tunnel identifier needs to be the travis job number